book/src/api-node.md

```diff
@@ -0,0 +1,37 @@
+# API Node
```

Comment:
Can we call this something less generic?  Maybe Explorer Node?

Reply:
Thinking more on it, I think there are two names we need to consider, the executable and the thread in Fullnode.  This proposal is for the executable, but it'd be useful for it also to propose the implementation strategy at a very high-level. The thread is currently named Entry Stream Stage. First, I'd suggest that "Stage" should be replaced with "Service".  Also, I think "Entry" can be replaced with "Block", which would lead to a natural user-facing name for the executable, `solana-blockstreamer`.

Reply:
so yes, @rob-solana, we may end up with Blockstore, Blocktree, Blockchain, and Blockstreamer. :)